src/Cassandra.IntegrationTests/Core/ExceptionsTests.cs

```diff
@@ -15,67 +15,90 @@
 //
 
 using System.Diagnostics;
-using System.Security;
 using Cassandra.IntegrationTests.TestBase;
 using Cassandra.IntegrationTests.TestClusterManagement;
 using NUnit.Framework;
 using System;
 using System.Collections.Generic;
+using System.Globalization;
 using System.Linq;
 using System.Net;
 using System.Threading;
-#if !NETCORE
-using System.Security.Permissions;
-#endif
 using Cassandra.Tests;
 
 namespace Cassandra.IntegrationTests.Core
 {
-    [TestFixture, Category("long")]
+    [TestFixture, Category("short")]
     public class ExceptionsTests : TestGlobals
     {
+        private ITestCluster _testCluster;
+        private ISession _session;
+        private const string KeyspaceNameWriteFailName = "ks_wfail";
+        private string _keyspace;
+        private string _table;
+
+        [OneTimeSetUp]
+        public void TestFixtureSetup()
+        {
+            _testCluster = TestClusterManager.CreateNew(2, new TestClusterOptions
```

Comment:
These tests were built before scassandra / simulacron integration.
I think we should avoid trying to test it with ccm and do it with simulacron.